pkgs/applications/video/droidcam/default.nix

```diff
@@ -1,8 +1,17 @@
 { lib, stdenv, fetchFromGitHub
 , ffmpeg, libjpeg_turbo, gtk3, alsa-lib, speex, libusbmuxd, libappindicator-gtk3
-, pkg-config
+, pkg-config, makeDesktopItem
 }:
 
+let
+  desktopItem = makeDesktopItem {
```

Comment:
There is actually a [desktop file](https://github.com/dev47apps/droidcam/blob/00846d3b1547aa80877c42108b7950d94c2aa8b4/droidcam.desktop) in the project. We might want to use that and just patch the paths.